resilience4j-ratelimiter/src/main/java/io/github/resilience4j/ratelimiter/RateLimiter.java

```diff
@@ -318,7 +318,23 @@ static void waitForPermission(final RateLimiter rateLimiter) {
 	 * @return {@code true} if a permit was acquired and {@code false}
 	 * if waiting timeoutDuration elapsed before a permit was acquired
 	 */
-	boolean acquirePermission();
+	default boolean acquirePermission() {
+		return acquirePermission(1);
+	}
+
+	/**
+	 * Acquires a permission of a given weight from this rate limiter, blocking until one is available, or the thread is interrupted.
+	 * Maximum wait time is {@link RateLimiterConfig#getTimeoutDuration()}
+	 *
+	 * <p>If the current thread is {@linkplain Thread#interrupt interrupted}
+	 * while waiting for a permit then it won't throw {@linkplain InterruptedException},
+	 * but its interrupt status will be set.
+	 * 
+	 * @param weight weight of permission - use for systems where 1 call != 1 permission
+	 * @return {@code true} if a permit was acquired and {@code false}
+	 * if waiting timeoutDuration elapsed before a permit was acquired
+	 */
+	boolean acquirePermission(int weight);
```

Comment:
Can we rename `weight` to `permits`. Analogous to `https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/Semaphore.html#acquire(int)`?